src/Microsoft.Framework.PackageManager/Bundle/BundleOperations.cs

```diff
@@ -17,15 +18,40 @@ public void Delete(string folderPath)
             FileOperationUtils.DeleteFolder(folderPath);
         }
 
-        public void Copy(string sourcePath, string targetPath)
+        public void Copy(IEnumerable<string> sourceFiles, string sourceDirectory, string targetDirectory)
         {
-            Copy(
-                sourcePath, 
-                targetPath, 
-                shouldInclude: _ => true);
-        }
+            sourceDirectory = PathUtility.EnsureTrailingSlash(sourceDirectory);
+            targetDirectory = PathUtility.EnsureTrailingSlash(targetDirectory);
+
+            foreach (var sourceFilePath in sourceFiles)
+            {
+                var fileName = Path.GetFileName(sourceFilePath);
+                Debug.Assert(fileName != null, "fileName != null");
```

Comment:
I don't want to change the existing behavior to complicate the case. The source files collection return from `Matcher` is always a absolute file paths collection.